CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Enhancements
 
-* None.
+* Prohibit to create or add objects of a class without properties to a Realm.
```

Comment:
Maybe this should go in the "Bugfixes" section? And it'd be good to elaborate here:

> Prohibit creating or adding objects of a class with no properties to a Realm. This previously would have silently failed to persist the object beyond the current transaction.